tests/Functional/PortabilityTest.php

```diff
@@ -79,22 +45,29 @@ public function testFullFetchMode(): void
         }
     }
 
-    public function testConnFetchMode(): void
+    /**
+     * @param array{int, list<string>} $expected
```

Comment:
Noticed during https://github.com/doctrine/dbal/pull/5252: this should be `list<string>`. it seems that PHPStan detects more things on 4.x